### ISSUE TITLE
v0.2.15 conflicts with fluent-plugin-td-monitoring

### ISSUE THREAD

**Issue by masutaka** [kaizenplatform/fluent-plugin-bigquery#63]
> STDERR: /opt/td-agent/embedded/lib/ruby/site_ruby/2.1.0/rubygems/specification.rb:2064:in `raise_if_conflicts': Unable to activate google-api-client-0.9.1, because httpclient-2.5.3.3 conflicts with httpclient (~> 2.7) (Gem::LoadError)

Reason:
td-agent 2.3.0 buildin plugin `fluent-plugin-td-monitoring` requires `httpclient [">= 2.4.0", "< 2.6.0"]`. But `fluent-plugin-bigquery` requires `httpclient '~> 2.7'` by  `google-api-client 0.9.1`.

**Comment by bryan-radiant** [comment]
We were having this same problem with fluent-plugin-bigquery.  My solution was this:

execute "Install fluentd-bigquery" do
   command "/opt/google-fluentd/embedded/bin/gem install fluent-plugin-bigquery -v 0.2.13; /opt/google-fluentd/embedded/bin/gem uninstall fluent-plugin-bigquery -v 0.2.15"
   not_if "/opt/google-fluentd/embedded/bin/gem list fluent-plugin-bigquery|grep 'bigquery (0.2.13)'"
end

Basically downgrade to 0.2.13 and everything works fine.

**Comment by masutaka** [comment]
Probably fixed by 2fe1a813207e3c9d792a2da00ee46b1f9cd74ed7
Thanks.
